### PR TITLE
[Task #51979] Theme user

### PR DESCRIPTION
**Pull request by thonght-3051**
## Related Tickets
- [#51979](https://edu-redmine.sun-asterisk.vn/issues/51979)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

### CODE REVIEW

**Comment by thonght-3051** [comment]
ready

**Review by dangthibichngocb — APPROVED** [review]

**Comment by manhnt-0960** [comment]
commented
chưa cập nhật redmine

**Comment by thonght-3051** [comment]
ready

File: app/views/layouts/dashboard.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= full_title yield :title %></title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "plugin/css/style", media: "all", "data-turbolinks-track": "reload" %>
```
Comment (huongnt-2545) [inline app/views/layouts/dashboard.html.erb:8]: sao link tag lại nhận từ file `style` thế em??

Comment (thonght-3051) [reply app/views/layouts/dashboard.html.erb:8]: Sample_app vẫn dùng  stylesheet_link_tag mà chị

**Comment by thonght-3051** [comment]
ready

**Pull request merged**